sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/ShuffleExchangeExec.scala

```diff
@@ -280,7 +280,7 @@ object ShuffleExchangeExec {
           }
           // The comparator for comparing row hashcode, which should always be Integer.
           val prefixComparator = PrefixComparators.LONG
-          val canUseRadixSort = SparkEnv.get.conf.get(SQLConf.RADIX_SORT_ENABLED)
+          val canUseRadixSort = SQLConf.get.enableRadixSort
```

Comment:
@ueshin, BTW, for clarification, it does read the configuration but does not respect when the configuration is given to the session, right? I think we don't need to backport this through all other branches.

Reply:
Ah, yes, to be exact, if users specified the config to `SparkConf` before Spark ran, it could be read.
I'd leave which branch we should backport to to you and other reviewers. @jiangxb1987 @cloud-fan

Reply:
It's a small bug fix, so no need to backport to all the branches. I think 2.4 is good enough